src/components/Main/NavCircle/NavCircle.tsx

```diff
@@ -1,19 +1,173 @@
+        />
+        <mask id="prefix__c">
+          <g filter="url(#prefix__a)">
+            <path fillOpacity={0.07} d="M0 0h48v48H0z" />
+          </g>
+        </mask>
+        <mask id="prefix__e">
+          <g filter="url(#prefix__a)">
+            <path fillOpacity={0.05} d="M0 0h48v48H0z" />
+          </g>
+        </mask>
+        <filter
+          id="prefix__a"
+          filterUnits="objectBoundingBox"
+          x="0%"
+          y="0%"
+          width="100%"
+          height="100%"
+        >
+          <feColorMatrix
+            in="SourceGraphic"
+            values="0 0 0 0 1 0 0 0 0 1 0 0 0 0 1 0 0 0 1 0"
+          />
+        </filter>
+      </defs>
+      <path
+        d="M42 45.477a.5.5 0 01-.809.394L33 39l.781 2.344a.5.5 0 01-.476.656H10a5 5 0 01-5-5V11a5 5 0 015-5h27a5 5 0 015 5zm0 0"
+        fill="#003046"
+      />
+      <path
+        d="M32.629 17.297a.358.358 0 00-.09-.098c-.383-.281-2.805-1.988-5.629-2.199l-.27.55c2.782.669 4.051 1.641 5.38 2.829C29.73 17.21 27.46 16 23.5 16s-6.23 1.21-8.52 2.379c1.329-1.188 2.848-2.27 5.38-2.828L20.09 15c-2.957.285-5.262 1.926-5.63 2.2a.324.324 0 00-.085.093c-.336.512-2.867 4.61-3.363 12.145a.426.426 0 00.11.316c2.554 2.805 6.16 3.187 7.058 3.238a.386.386 0 00.336-.16l.797-1.062c-1.57-.54-3.364-1.508-4.903-3.27 1.84 1.379 4.61 2.5 9.09 2.5 4.48 0 7.25-1.121 9.09-2.5-1.54 1.762-3.328 2.73-4.899 3.27l.793 1.062a.395.395 0 00.336.16c.903-.05 4.504-.433 7.063-3.238a.44.44 0 00.105-.317c-.496-7.53-3.027-11.628-3.36-12.14zM20 28c-1.102 0-2-1.121-2-2.5s.898-2.5 2-2.5c1.102 0 2 1.121 2 2.5s-.898 2.5-2 2.5zm7 0c-1.102 0-2-1.121-2-2.5s.898-2.5 2-2.5c1.102 0 2 1.121 2 2.5s-.898 2.5-2 2.5zm0 0"
+        fill="#fff"
+      />
+      <use xlinkHref="#prefix__b" mask="url(#prefix__c)" />
+      <use xlinkHref="#prefix__d" mask="url(#prefix__e)" />
+      </a>
+    </svg>
+  )
+}
+
+const Circle = () => {
+  return (
+<svg  
+      xmlns="http://www.w3.org/2000/svg"
+      width="100%"
+      height="100%"
+      viewBox="0 0 303 303"
+      overflow="visible"
+      
+    >
+      <circle
+        cx={151.5}
+        cy={151.5}
+        r={125}
+        fill="none"
+        stroke="#43d6b9"
+        strokeWidth={3}
+        strokeMiterlimit={10}
+      />
+    </svg>
+
+  )
+}
 
 function NavCircle() {
   return (
-    <div>
-      <a href="#" target="_blank">the gap</a>
-      <a href="#" target="_blank">the mission</a>
-      <a href="#" target="_blank">the team</a>
-      <a href="#" target="_blank"><img src="#" alt="Instagram" /></a>
-      <a href="#" target="_blank"><img src="#" alt="Discord" /></a>
+    <div id = "navCircle" className = "image sticky-top float-right " >
+      <Circle />
```

Comment:
is it possible for you to make Circle a div where all the links live, that way you can make the Circle absolute, but all the links can simply be in a row